TwitchPlaysAssembly/Src/UI/TwitchMessage.cs

```diff
@@ -20,7 +20,18 @@ private void Awake()
 	{
 		_messageBackground = GetComponent<Image>();
 		_messageText = GetComponentInChildren<Text>();
-		_messageBackground.color = NormalColor;
+	}
+	
+	private void Start()
+	{
+		if (!TwitchPlaySettings.data.DarkMode) {
+			_messageBackground.color = NormalColor;
+			_messageText.color = new Color32(0x4F, 0x4F, 0x4F, 0xFF);
+		}
+		else {
+			_messageBackground.color = new Color32(0x3A, 0x3A, 0x3D, 0xFF);
+			_messageText.color = new Color32(0xEF, 0xEF, 0xEC, 0xFF);
+		};
```

Comment:
Minor but braces need to be on separate line and that semicolon on the last line is unnecessary.